src/translations/lang/en.json

```diff
@@ -1016,6 +1016,7 @@
     "VISIT_KB": "Visit the Knowledge Base",
     "BUSY_BOTTOM_GET_WEB3": "Get $app",
     "BUSY_BOTTOM_METAMASK_1": "MetaMask Instructions",
-    "EXPORT_MOBILE_NOTICE": "Some mobile dapp browsers prevent downloading files - in that case, you can copy the above text manually."
+    "EXPORT_MOBILE_NOTICE": "Some mobile dapp browsers prevent downloading files - in that case, you can copy the above text manually.",
+    "VIEW_ONLY_ADDR_DISCLAIMER": "Adding a \"view-only\" account means that you will not be able to send a transaction from it. If you want the ability to send from this account, please choose another method of connection on the [Add Account](/add-account) screen. Or, you can add it as \"view-only\" for now and [fully connect it at another time](https://support.mycrypto.com/how-to/accessing-wallet/how-does-view-address-work)."
```

Comment:
We no longer add links in the translation files.
Prefer `getKBHelpArticle`

Reply:
How do I use `getKBHelpArticle` inline with the text? I should implement `ResourceLink` component twice and have 2 different `lang/en.json` entries?

Reply:
The translate function uses variable interpolation so we can call it with:
```
<>
  {
    translate('VIEW_ONLY_ADDR_DISCLAIMER', { 
	    $link: getKBHelpArticle(KB_HELP_ARTICLE.HOW_DOES_VIEW_ADDRESS_WORK)
    })
  }
</>
```

and within the `lang/en.json` we define the string as:
```
"VIEW_ONLY_ADDR_DISCLAIMER": "Adding a \"view-only\" account means that you will not be able to send a transaction from it. If you want the ability to send from this account, please choose another method of connection on the [Add Account](/add-account) screen. Or, you can add it as \"view-only\" for now and [fully connect it at another time]($link).
```

Reply:
Thanks! Amended in https://github.com/MyCryptoHQ/MyCrypto/pull/3858/commits/05557b64d4e0ea4bf9baf2f6461097441bca74ad